src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -1046,6 +1046,16 @@ else if (block instanceof BlockFlowerPot)
             }
         }
 
+        if(block instanceof BlockChest && job instanceof JobBuilder)
+        {
+            BlockPos buildingLocation = ((JobBuilder) job).getWorkOrder().getBuildingLocation();
+            AbstractBuilding building = this.getOwnBuilding().getColony().getBuilding(buildingLocation);
+            if(building instanceof AbstractBuilding)
+            {
+                ((AbstractBuilding) building).addContainerPosition(pos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unnecessary cast to "AbstractBuilding". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1905)